addons/sale_crm/models/crm_lead.py

```diff
@@ -10,23 +10,27 @@ class CrmLead(models.Model):
     _inherit = 'crm.lead'
 
     sale_amount_total = fields.Monetary(compute='_compute_sale_amount_total', string="Sum of Orders", help="Untaxed Total of Confirmed Orders", currency_field='company_currency')
-    sale_number = fields.Integer(compute='_compute_sale_amount_total', string="Number of Quotations")
+    quotation_count = fields.Integer(compute='_compute_sale_amount_total', string="Number of Quotations")
+    sale_order_count = fields.Integer(compute='_compute_sale_amount_total', string="Number of Sale Orders")
     order_ids = fields.One2many('sale.order', 'opportunity_id', string='Orders')
 
     @api.depends('order_ids')
     def _compute_sale_amount_total(self):
         for lead in self:
             total = 0.0
-            nbr = 0
+            quot_cnt = 0
```

Comment:
It is always a good idea to use meaningful variable name. It does not take a lot of additional time to write quotation_count and sale_order_count :) .